src/main/java/wolox/training/repositories/BookRepository.java

```diff
@@ -0,0 +1,10 @@
+package wolox.training.repositories;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import wolox.training.model.Book;
+
+public interface BookRepository extends JpaRepository<Book, Long> {
+
+    Book findByAuthorIgnoreCase(String author);
```

Comment:
I think you can use Optional<Book> here in order to send back an Optional and prevent null pointer exceptions. Might need a change regarding the declared superclass, I don't remember it fully.

Reply:
Yes! I just try it and it works fine :+1: 
Done!